tests/test_client.py

```diff
@@ -0,0 +1,273 @@
+import pytest
+import requests
+from src.client import Client
```

Comment:
Shouldn't the tests dir be inside `src`? Then you can just do `from .client import Client`